src/tests/functions/gif/test_gif.py

```diff
@@ -52,3 +64,50 @@ def test_gif_empty_params():
     result = gif(request_mock)
 
     assert result == DEFAULT_GIF
+
+
+def test_success_200():
+    """Test success api call"""
+    api_request_json = """{"results": [
+                {
+                    "media_formats": {
+                    "gif": {
+                        "url": "https://c.tenor.com/HvtIOMNuwhYAAAAC/wat-what.gif",
+                        "duration": 0,
+                        "preview": "",
+                        "dims": [
+                        436,
+                        498
+                        ],
+                        "size": 7986816
+                    }
+                    }
+                }
+                ]
+            }"""
+
+    result = "https://c.tenor.com/HvtIOMNuwhYAAAAC/wat-what.gif"
```

Comment:
`result` vs `expected_result` is not the same
The `result` var should be what comes back from the function call